web/setups/feedback.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright (c) 2017 University of Illinois, NCSA.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the
+ * University of Illinois/NCSA Open Source License
+ * which accompanies this distribution, and is available at
+ * http://opensource.ncsa.illinois.edu/license.html
+ */
+
+// feed back page
```

Comment:
I removed that page form this pull request. will send in future request when done.